iam/policy-templates/ci-cd.json

```diff
@@ -162,7 +162,11 @@
         "arn:aws:sqs:*:$account_id:dss-notify-test-*",
         "arn:aws:sqs:*:$account_id:dss-notify-dev-*",
         "arn:aws:sqs:*:$account_id:dss-notify-integration-*",
-        "arn:aws:sqs:*:$account_id:dss-notify-staging-*"
+        "arn:aws:sqs:*:$account_id:dss-notify-staging-*",
+        "arn:aws:sqs:*:$account_id:dss-notify-v2-test-*",
```

Comment:
we can remove the `-*` after the stage, we also should add the event relay sqs in here; it looks something like `dss-notify-v2-event-relay-dev`